<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="checkButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/images/chaozhao.png);
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="DateLcdNumber">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLCDNumber{
	font: 75 9pt &quot;Arial&quot;;
border-radius:20px;
}</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="TimeLcdNumber">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Unicode MS</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLCDNumber{
	border-radius:30px;
}</string>
    </property>
    <property name="digitCount">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="IndexLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial Rounded MT Bold&quot;;</string>
    </property>
    <property name="text">
     <string>感冒指数</string>
    </property>
   </widget>
   <widget class="QLabel" name="YesterdayLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>昨天</string>
    </property>
   </widget>
   <widget class="QLabel" name="YesterdayHighLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>high</string>
    </property>
   </widget>
   <widget class="QLabel" name="TodayLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>今天</string>
    </property>
   </widget>
   <widget class="QLabel" name="TodayHIghLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>high</string>
    </property>
   </widget>
   <widget class="QLabel" name="TomorrowLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>明天</string>
    </property>
   </widget>
   <widget class="QLabel" name="TomorrowHIghLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>270</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>high</string>
    </property>
   </widget>
   <widget class="QLabel" name="AfterTomorrowLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>120</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>后天</string>
    </property>
   </widget>
   <widget class="QLabel" name="AfterTomorrowHIghLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>270</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>high</string>
    </property>
   </widget>
   <widget class="QLabel" name="CityLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>城市名</string>
    </property>
   </widget>
   <widget class="QLabel" name="TemperatureLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>35</red>
          <green>35</green>
          <blue>35</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>35</red>
          <green>35</green>
          <blue>35</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="0">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>温度</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="GanmaoLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>感冒指数提醒</string>
    </property>
   </widget>
   <widget class="QLabel" name="YesterdayTypeText">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>天气类型</string>
    </property>
   </widget>
   <widget class="QLabel" name="TodayTypeText">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>天气类型</string>
    </property>
   </widget>
   <widget class="QLabel" name="TomorrowTypeText">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>天气类型</string>
    </property>
   </widget>
   <widget class="QLabel" name="AfterTomorrowTypeText">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>170</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>天气类型</string>
    </property>
   </widget>
   <widget class="QLabel" name="YesterdayLowLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>low</string>
    </property>
   </widget>
   <widget class="QLabel" name="TodayLowLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>310</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>low</string>
    </property>
   </widget>
   <widget class="QLabel" name="TomorrowLowLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>310</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>low</string>
    </property>
   </widget>
   <widget class="QLabel" name="AfterTomorrowLowLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>310</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>low</string>
    </property>
   </widget>
   <widget class="QLabel" name="YesterdayTypeLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>天气类型图</string>
    </property>
   </widget>
   <widget class="QLabel" name="TodayTypeLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>天气类型图</string>
    </property>
   </widget>
   <widget class="QLabel" name="AfterTomorrowTypeLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>200</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>天气类型图</string>
    </property>
   </widget>
   <widget class="QLabel" name="TomorrowTypeLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>200</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>天气类型图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>31</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-color: rgb(73, 164, 212);
	background-color: rgba(255, 255, 255, 0);
	background-image: url(:/new/prefix1/images/update.png);
	border-style:solid;
	border-top-left-radius:1px;
	border-top-right-radius:1px;
}
QPushButton:Pressed{
	background-image: url(:/new/prefix1/images/update_on.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="minpushButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/images/min.png);
background-color: rgba(255, 255, 255, 0);
border-top-color: rgba(255, 255, 255, 0);
border-style:solid;
	border-top-left-radius:1px;
	border-top-right-radius:1px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="closepushButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/images/close.png);
background-color: rgba(255, 255, 255, 0);
background-color: rgba(255, 255, 255, 0);
border-style:solid;
	border-top-left-radius:1px;
	border-top-right-radius:1px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="InputLineEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
border:1px solid rgb(180, 180, 180);
background: rgb(230,230,230);

border-radius: 4px;

}
QLineEdit::hover
{

 //字体的颜色

  color:green
  border-color:rgb(50,480,40);
  background-color:green;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="aqi">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>350</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pm25Text">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PM2.5指数:</string>
    </property>
   </widget>
   <widget class="QLabel" name="airQualityTextlabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>350</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>空气质量</string>
    </property>
   </widget>
   <widget class="QLabel" name="airQuality">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>350</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="YesterdayWeek">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>星期（昨）</string>
    </property>
   </widget>
   <widget class="QLabel" name="TomorrowWeek">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>星期（明）</string>
    </property>
   </widget>
   <widget class="QLabel" name="AfterTomorrowWeek">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>121</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>星期（后）</string>
    </property>
   </widget>
   <widget class="QLabel" name="FengxiangLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Anonymous</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>风向</string>
    </property>
   </widget>
   <widget class="QLabel" name="FengliLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>风力</string>
    </property>
   </widget>
   <zorder>DateLcdNumber</zorder>
   <zorder>TimeLcdNumber</zorder>
   <zorder>IndexLabel</zorder>
   <zorder>YesterdayLabel</zorder>
   <zorder>YesterdayHighLabel</zorder>
   <zorder>TodayLabel</zorder>
   <zorder>TodayHIghLabel</zorder>
   <zorder>TomorrowLabel</zorder>
   <zorder>TomorrowHIghLabel</zorder>
   <zorder>AfterTomorrowLabel</zorder>
   <zorder>AfterTomorrowHIghLabel</zorder>
   <zorder>CityLabel</zorder>
   <zorder>TemperatureLabel</zorder>
   <zorder>GanmaoLabel</zorder>
   <zorder>YesterdayTypeText</zorder>
   <zorder>TodayTypeText</zorder>
   <zorder>TomorrowTypeText</zorder>
   <zorder>AfterTomorrowTypeText</zorder>
   <zorder>YesterdayLowLabel</zorder>
   <zorder>TodayLowLabel</zorder>
   <zorder>TomorrowLowLabel</zorder>
   <zorder>AfterTomorrowLowLabel</zorder>
   <zorder>YesterdayTypeLabel</zorder>
   <zorder>TodayTypeLabel</zorder>
   <zorder>AfterTomorrowTypeLabel</zorder>
   <zorder>TomorrowTypeLabel</zorder>
   <zorder>pushButton</zorder>
   <zorder>minpushButton</zorder>
   <zorder>closepushButton</zorder>
   <zorder>InputLineEdit</zorder>
   <zorder>checkButton</zorder>
   <zorder>aqi</zorder>
   <zorder>pm25Text</zorder>
   <zorder>airQualityTextlabel</zorder>
   <zorder>airQuality</zorder>
   <zorder>YesterdayWeek</zorder>
   <zorder>TomorrowWeek</zorder>
   <zorder>AfterTomorrowWeek</zorder>
   <zorder>FengxiangLabel</zorder>
   <zorder>FengliLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>minpushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>520</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closepushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>565</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
